tasks/preflight.yml

```
@@ -25,19 +25,26 @@
         promtail_version: "{{ _latest_release.json.tag_name[1:] }}"
   when:
     - promtail_version == "latest"
-  delegate_to: localhost
-  run_once: True
-
 
-- name: "Get checksum list"
-  set_fact:
-    __promtail_checksums: "{{ lookup('url', 'https://github.com/grafana/loki/releases/download/v' + promtail_version + '/SHA256SUMS', wantlist=True) | list }}"
-  run_once: True
+- name: Test if zip local archive exists
+  stat:
+    path: "{{ promtail_binary_local_dir }}/{{ promtail_version }}_promtail-linux-{{ go_arch }}.zip"
+  register: promtail_binary_local_archive
   delegate_to: localhost
+  run_once: true
```

nitpick from the linter

> ERROR: [ANSIBLE0014] Standard 'Literal bools should start with a capital letter' not met:
> ... tasks/preflight.yml:34: literal bools should be written as `True/False` or `yes/no`

```suggestion
  run_once: True
```